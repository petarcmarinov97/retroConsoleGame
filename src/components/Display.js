import React, { Component } from 'react';
import "../styles/Display.css";
import Game from './Game';
import ScoreBoard from './ScoreBoard';

export default class Display extends Component {
    render() {
        return <div className="display">
            <>
                <Game />
                <ScoreBoard />
            </>
        </div>;
    }
}
