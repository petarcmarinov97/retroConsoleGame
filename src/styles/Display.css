div.display {
    display: flex;
    max-height: 400px;
    max-width: 360px;
    height: 400px;
    width: 310px;
    background-color: #9fae87;
    border: 2px inset #a18803;
    border-radius: 3px;
}