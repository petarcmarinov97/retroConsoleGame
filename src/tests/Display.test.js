import React from 'react';
import { shallow } from 'enzyme';
import { shallowToJson } from "enzyme-to-json";
import Display from '../components/Display';

describe('Display tests', () => {

    let wrapper;

    beforeEach(() => {
        wrapper = shallow(
            <Display />);
    })

    describe('Snapshots', () => {
        it("should render correctly", () => {
            expect(shallowToJson(wrapper)).toMatchSnapshot();
        })
    })
})