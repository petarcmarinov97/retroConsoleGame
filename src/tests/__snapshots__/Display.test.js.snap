// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Display tests Snapshots should render correctly 1`] = `
<div
  className="display"
>
  <Game />
</div>
`;
